src/domains/dashboard/pages/Dashboard/Dashboard.Empty.blocks.tsx

```diff
@@ -60,36 +60,39 @@ export const DashboardSetupAddressCard = ({
 	image,
 	buttonText,
 	onClick,
-	variant = "primary",
 }: {
 	title: string;
 	description: string;
 	image: string;
 	buttonText: string;
 	onClick: () => void;
-	variant?: "primary" | "secondary";
 }) => (
-	<div
-		className={cn("rounded border border-theme-secondary-300 p-4 dark:border-theme-dark-700 md:rounded-xl md:p-8", {
-			"bg-theme-primary-100 dark:bg-theme-dark-800": variant === "primary",
-		})}
+	<button
+		className={cn(
+			"group rounded border border-theme-secondary-300 bg-white p-4 transition-all hover:border-theme-primary-50 hover:bg-theme-primary-50 dark:border-theme-dark-700 dark:bg-theme-dark-900 dark:hover:border-theme-dark-950 dark:hover:bg-theme-dark-950 md:rounded-xl md:p-8",
+		)}
+		type="button"
+		onClick={onClick}
 	>
 		<h3
-			className={cn("md:leading-7.5 mb-2 text-lg font-semibold leading-4 md:text-2xl", {
-				"text-theme-primary-900 dark:text-theme-dark-50": variant === "primary",
-				"text-theme-secondary-900 dark:text-theme-dark-50": variant === "secondary",
-			})}
+			className={cn(
+				"md:leading-7.5 mb-2 text-lg font-semibold leading-4 text-theme-secondary-900 group-hover:text-theme-primary-900 dark:text-theme-dark-50 md:text-2xl",
+			)}
 		>
 			{title}
 		</h3>
 		<p className="mb-6 text-sm leading-5 text-theme-secondary-700 dark:text-theme-dark-200">{description}</p>
 		<div className="mx-auto my-6 flex max-w-56 justify-center md:max-w-full">
 			<Image name={image} />
 		</div>
-		<Button variant={variant} className="w-full" onClick={onClick}>
+		<Button
+			variant="secondary"
+			className="w-full dark:border dark:border-theme-dark-600 dark:hover:border-theme-dark-navy-700 dark:hover:bg-theme-dark-navy-700"
```

Comment:
make the button part of the hover of the card as well, so it doesn't have its own hover trigger to avoid this

<img width="440" alt="image" src="https://github.com/user-attachments/assets/c7b93962-487a-4867-8939-b985271ce713" />